pallets/rc-migrator/src/staking/nom_pools.rs

```diff
@@ -19,8 +19,17 @@
 use super::nom_pools_alias as alias;
 use crate::{types::*, *};
 use alias::{RewardPool, SubPools};
-use pallet_nomination_pools::{BondedPoolInner, ClaimPermission, PoolId, PoolMember};
-use sp_runtime::Perbill;
+use frame_support::traits::{ConstU32, Get};
+use pallet_nomination_pools::{
+	BondedPoolInner, ClaimPermission, CommissionChangeRate, CommissionClaimPermission, PoolId,
+	PoolMember, PoolRoles, PoolState,
+};
+use sp_runtime::{
+	traits::{One, Zero},
+	Perbill, Saturating,
+};
+use sp_staking::EraIndex;
+use sp_std::fmt::Debug;
```

Comment:
these imports if only used for std items should trigger warnings. 
you could do something like

``` rust 
#[cfg(feature = "std")]
pub mod tests {
  use ...; // all your imports 

  // all your new types 
  pub struct GenericCommission
  
  // the type implementing test trait 
  pub struct NomPoolsMigratorTest
}
```